src/core/core.datasetController.js

```diff
@@ -236,6 +238,33 @@ helpers.extend(DatasetController.prototype, {
 		me.resyncElements();
 	},
 
+	/**
+	 * Returns the merged user-supplied and default dataset-level options
+	 * @private
+	 */
+	_configure: function() {
+		var me = this;
+		var dataset = me.getDataset();
+		var datasetDefaults = defaults.global.datasets[me._type];
+		var userOpts = {};
+		var keys = Object.keys(dataset);
+		var i, ilen, key;
+		for (i = 0, ilen = keys.length; i < ilen; i++) {
+			key = keys[i];
+			if (key !== '_meta' && key !== 'data') {
+				userOpts[key] = helpers.clone(dataset[key]);
+			}
+		}
+		datasetDefaults = datasetDefaults !== undefined ? helpers.clone(datasetDefaults) : {};
```

Comment:
```js
datasetDefaults = helpers.clone(datasetDefaults) || {};
```
or
```js
var datasetDefaults = helpers.clone(defaults.global.datasets[me._type]) || {};
```
at the beginning as `clone(undefined)` returns `undefined`.

Reply:
done